### PR TITLE
Check in DatePicker topic

### PR DESCRIPTION
**Pull request by agoldenbaum**

### CODE REVIEW

File: doc/en/components/date-picker.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+mentionedTypes: []
+---
+# $PlatformShort$ Date Picker Overview
+
+The Date Picker component allows users to use a drop-down calendar UI allowing the intuitive selection of a day, month and year. This can be helpful when an application user needs to select specific dates, and multiple editors can be combined to create a date-range UI.
+
+## Usage
+
+The $PlatformShort$ Date Picker editor component consists of three default parts - the text area, a calendar drop-down button, and a clear button. The text area will display the selected date, stored in the `Value` property of the editor.
+
```
Comment (pamelabrasil) [inline doc/en/components/date-picker.md:14]: add bullets here to describe each area 
Text Area- the text area will display the selected date, stored in the value property of the editor
Calendar -
Clear button -

File: doc/en/components/date-picker.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ModuleManager.register(
+    IgcDatePickerModule
+);
+```
+
+<div class="divider--half"></div>
+
+## Code Snippet
+
```
Comment (pamelabrasil) [inline doc/en/components/date-picker.md:70]: explain the code snippet with a description and add some more things to - for example setting specific properties with comments

File: doc/en/components/date-picker.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: $PlatformShort$ Date Picker | Data Visualization Tools | Infragistics
+_description: Infragistics' $PlatformShort$ date picker component helps your users select dates. Improve your graphs and visualization with Ignite UI for  $PlatformShort$!
+_keywords: $PlatformShort$ date picker, drop down, $ProductName$, Infragistics
+mentionedTypes: []
```
Comment (HUSSAR-mtrela) [inline doc/en/components/date-picker.md:5]: in the `mentionedTypes` array, add name of class(es) that have implementation of properties mentioned in this topic

File: doc/en/components/date-picker.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+IgrDatePickerModule.register();
+```
+
+```ts
+import { IgcDatePickerModule } from 'igniteui-webcomponents-inputs';
+import { ModuleManager } from 'igniteui-webcomponents-core';
+
+ModuleManager.register(
+    IgcDatePickerModule
+);
```
Comment (HUSSAR-mtrela) [inline doc/en/components/date-picker.md:64]: change it to single line

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**